docs/source/fundamentals.md

```diff
@@ -101,29 +99,155 @@ Once a certain degree of interoperability has been reached, other interesting co
   Being able to know the semantic meaning of an instance and hence its connection to other concepts
   is one of the principles of SimPhoNy. For that, ontologies play a major role.
 
-#### Requirement simplification
-  Since we know what a user means from the semantic approach, we can use this to automatise and simplify
-  the setup and initialisation of processes using default settings.
+### Ontology
+```eval_rst
+.. important::
+   An ontology is a formal specification of a shared conceptualization.  `[Borst, 1997]
+   <https://research.utwente.nl/en/publications/construction-of-engineering-ontologies-for-knowledge-sharing-and->`_ .
+
+```
+
+Let's look at the individual components of this definition, starting from the end.
+ - _Conceptualization_, an ontology will work on the ideas and relationships in an area of interest.
+ - _Shared_, the ideas and concepts are perceived and agreed by multiple people.
+ - _Specification_, it will define and describe them in detail, following some predetermined rules and format.
+ - _Formal_, meaning it will follow a machine readable syntax.
+
+In a simpler way, an ontology can be seen as the definition of concepts relevant to a given domain, 
+as well as the relationships between them, in a way that a machine can interpret it.
+
+For a deeper, more detailed analysis of the definition, refer to [[Guarino, 2009]](http://dx.doi.org/10.1007/978-3-540-92673-3_0).
+
+Ontologies are more elaborated than taxonomies in that they can include multiple kinds of relationships
+(not just parent-child) between complex concepts in big domains.
+
+#### EMMO
+TODO
+### CUDS
+Cuds, or Common Universal Data Structure, is the ontology compliant data format of OSP-core:
```

Comment:
it's probably a good idea to be consistent with how we write CUDS / Cuds. I would go for CUDS.

Reply:
I left CUDS only for the header, thought in general Cuds would be easier to write.
But I'm not committed to any particular spelling